config/config.go

```diff
@@ -134,6 +134,10 @@ func LoadFile(filename string, agentMode bool, logger *slog.Logger) (*Config, er
 		}
 	}
 
+	if cfg.GlobalConfig.MetricNameValidationScheme == "legacy" && cfg.OTLPConfig.TranslationStrategy == NoUTF8EscapingWithSuffixes {
+		return nil, errors.New("otlp translation strategy NoUTF8EscapingWithSuffixes is not allowed when UTF8 is disabled")
+	}
```

Comment:
This check should be moved into `Load`, for completeness.

Reply:
It's not strange @ArthurSens. The other validation in `LoadFile` is dependent on `agentMode`, which is a parameter only to `LoadFile` and not `Load`.

Reply:
moved...